apps/src/blockly/addons/cdoSerializationHelpers.js

```diff
@@ -314,3 +314,45 @@ export function getCombinedSerialization(
   );
   return combinedSerialization;
 }
+
+export function convertFunctionsXmlToJson(functionsXml) {
```

Comment:
can you add comments on this function and `appendProceduresToState`?

Reply:
Will do!